frontend/src/lib/components/LemonActionableTooltip/LemonActionableTooltip.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { Placement } from '@floating-ui/react-dom-interactions'
+import { Popup } from 'lib/components/Popup/Popup'
+import { IconOpenInNew } from 'lib/components/icons'
+import { CloseOutlined, LeftOutlined, RightOutlined } from '@ant-design/icons'
```

Comment:
Can we use `IconClose`, `IconChevronLeft`, and `IconChevronRight` instead?

Reply:
<img width="485" alt="Screenshot 2022-08-01 at 10 29 15" src="https://user-images.githubusercontent.com/7335343/182118604-cec802b2-51a4-4965-8a6b-8a5e1fb7e1c0.png">

Great thanks, removed them!